src/lib/scripts/package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Should this file be deleted?

Reply:
If I remove that file, I get this error when I run `npm run sandbox`: 

Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: C:\Users\jones\Code\sil\appbuilder-pwa\src\lib\scripts\thaw.ts

I found that creating a package.json in the scripts folder resolved this issue, even if the JSON was empty.  I wish I could leave a comment about this, but I'm not sure if comments are possible in JSON.

Reply:
Very odd!

Reply:
I just added a "description" field that explains the issue